depends/packages/qt.mk

```diff
@@ -70,19 +71,35 @@ $(package)_config_opts += -system-zlib
 $(package)_config_opts += -static
 $(package)_config_opts += -silent
 $(package)_config_opts += -v
+$(package)_config_opts += -no-feature-bearermanagement
+$(package)_config_opts += -no-feature-colordialog
+$(package)_config_opts += -no-feature-commandlineparser
```

Comment:
Does this remove the ability to add qt-specific runtime args to bitcoin-qt ?

Reply:
It shouldn't do. We don't use any [`QCommandLineParser`](https://doc.qt.io/qt-5/qcommandlineparser.html) functionality, and it is actually another layer on top of any command line argument handling done by [`QCoreApplication`](https://doc.qt.io/qt-5/qcoreapplication.html#accessing-command-line-arguments).

I have tested passing through some qt specific options, such as `-qwindowtitle` and they still seem to work:

![Hello World](https://user-images.githubusercontent.com/863730/61760598-a8fd9400-adfe-11e9-848e-a085e76f18cb.png)